packages/core/src/actions/smartAccount/types.ts

```diff
@@ -26,6 +27,18 @@ export type SendUserOperationParameters<
   overrides?: UserOperationOverrides;
 } & GetAccountParameter<TAccount>;
 
+export type SignUserOperationParameters<
+  TAccount extends SmartContractAccount | undefined
+> = {
+  uoStruct: UserOperationStruct;
+} & GetAccountParameter<TAccount>;
+
+export type SendRawUserOperationParameters<
+  TAccount extends SmartContractAccount | undefined
+> = {
+  request: UserOperationRequest;
+} & GetAccountParameter<TAccount>;
```

Comment:
this is already defined on the `bundlerClient` as part of `bundlerActions` and already added to the smartAccountClient
```suggestion

```

Reply:
ditto: my comment above. will change the name of this added func.